pkg/shipyard/podspec_generator.go

```diff
@@ -12,6 +12,11 @@ import (
 	"k8s.io/kubernetes/pkg/api"
 )
 
+const (
```

Comment:
These should be environment variables because these values are configurable at the `k8s-router` level so we should make them configurable here as well.